pkg/network/sftp.go

```diff
@@ -193,20 +193,3 @@ func ReadFileFromSFTPUsingPrivateKey(user, privateKey, address, srcFile, dstFile
 
 	return ReadFileFromSFTPUsingConfig(config, address, srcFile, dstFile)
 }
-
-func ReadFileFromLegacySFTP(usr, password, address, srcFile, dstFile string) (int64, error) {
```

Comment:
Let's not delete it instead make it more dynamic that we take cipher as a param.